server/settings.py

```diff
@@ -203,3 +203,24 @@
 
 # Disable purging of publish queue
 PUBLISH_QUEUE_EXPIRY_MINUTES = 0
+
+SCHEMA_UPDATE = {
+    "archive": {
+        "extra": {
+            "type": "dict",
+            "schema": {},
+            "mapping": {
+                "type": "object",
+                "properties": {
+                    "university": {
+                        "properties": {
```

Comment:
I think it's just a text field in the `extra` so that won't have any name and qcode.
also would be good to set dynamic: false in the mapping so it won't add additional properties automatically

Reply:
alright, Thanks